docker/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM php:latest
```

Comment:
You can use an alpine image to speed up tests, something like this:

```dockerfile
ARG PHP_VERSION=7.2
FROM php:${PHP_VERSION}-alpine

RUN apk add --no-cache \
        autoconf \
        make \
        g++
RUN pecl install mongodb && docker-php-ext-enable mongodb
RUN set -o pipefail && curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
RUN echo -e '#!/bin/sh' >> /usr/local/bin/entrypoint.sh \
    && echo -e 'while ! nc -z ${MONGO_HOST:-mongo} ${MONGO_PORT:-9000}; do sleep 1; done' >> /usr/local/bin/entrypoint.sh \
    && echo -e 'exec "$@"' >> /usr/local/bin/entrypoint.sh \
    && chmod +x /usr/local/bin/entrypoint.sh
WORKDIR /docker
ENTRYPOINT ["/usr/local/bin/entrypoint.sh"]
CMD ["./vendor/bin/phpunit"]
```

You can specify a build argument to say which image to build.
I also added an entry-point to wait mongodb service to be up.

Reply:
Obviously you should change code where mongodb host is `localhost`, using environment variables `$MONGO_HOST` and `$MONGO_PORT` 